src/main/js/store/actions/globalConfiguration.js

```diff
@@ -152,6 +167,27 @@ export function refreshConfig(id) {
     };
 }
 
+export function getAllConfigs(descriptorName) {
+    return (dispatch, getState) => {
+        dispatch(fetchingConfig());
+        const { csrfToken } = getState().session;
```

Comment:
Are we storing the CSRF token as a cookie?

Reply:
The CSRF token is stored in the redux state for the application.  When you login we update the redux state with the CSRF token for the application.  It isn't stored as a cookie.